spec/models/migration_engine_spec.rb

```diff
@@ -326,6 +326,40 @@
           end
         end
 
+        context 'complex case with SLE 15 module dependencies' do
```

Comment:
Context name is duplicated from https://github.com/SUSE/rmt/pull/132/files#diff-b8fe386eae7950826d27f1d741fe4562L329 no?